include/aikido/control/InstantaneousTrajectoryExecutor.hpp

```diff
@@ -17,20 +16,24 @@ class InstantaneousTrajectoryExecutor : public TrajectoryExecutor
 {
 public:
   /// Constructor.
+  ///
   /// \param skeleton Skeleton to execute trajectories on.
   ///        All trajectories must have dofs only in this skeleton.
+  /// \param timestep The time period that each call to step() should simulate
   explicit InstantaneousTrajectoryExecutor(
-      ::dart::dynamics::SkeletonPtr skeleton);
+      ::dart::dynamics::SkeletonPtr skeleton,
+      std::chrono::milliseconds timestep);
```

Comment:
Doesn't `InstantenousTrajectoryExecutor` steps immediately to the end? Why do we need to set the timestep?

Reply:
You're right, I don't think we need to.